docs/snowflake-native-apps/event-sharing.md

```diff
@@ -0,0 +1,20 @@
+## Event Sharing
+
+Sharing the events is important to ensure that in case of issue, our team can investigate easily. The app will never share private or sensitive information, only errors that will allow our team to understand the problem.
```

Comment:
```suggestion
Sharing the events is important to ensure that in case of issue, our team can investigate easily. The app will never share private or sensitive information, only errors and diagnostic metrics that allow our team to understand the problem.
```